### DISCUSSION TITLE
Flutter & FHIR

### DISCUSSION THREAD

**Discussion by Dokotela** [google/android-fhir#423] [Show and tell]
I realize this is a project for native Android, but just in case it's helpful (and for self promotion ;-) I  thought I'd mention the work I've been doing in Flutter. I head an open-source collaborative called [FHIR-FLI](https://www.fhirfli.dev/) (Fast Healthcare Interoperability Resources with Flutter Library Integration), and I've written a number of libraries for FHIR that run in Flutter.

The primary github repo is here: https://github.com/MayJuun/fhir
And most of the packages are also available on Pub.dev: https://pub.dev/publishers/fhirfli.dev/packages

I've tried to create basic packages for functionality that might be useful. So far this includes the basic FHIR classes, serializing, deserializing, etc. authentication/authorizing, making RESTful requests, storing offline encrypted data, and working with bulk FHIR data. 

With the authentication package I've been able to use Aidbox, HAPI, Logica, MiHIN, GCP Healthcare API and Azure. But right now it only works on Android & iOS, not yet for web.

I'm also working on developing a package that will run with Firestore.

As I said, I know this project is for native android, but since everything I've published so far will work on Android, I thought I'd go ahead and post it here as well.

**Comment by ajhmohammed** [comment]
Very interesting. Let me go through your work and contact you if I need any support.
